config/locales/api/en.yml

```diff
@@ -19,6 +19,22 @@ en:
     patients_enrolled: "How many new patients are enrolled at %{facility_name}?"
     all_time_patients: "All time: %{count} patients"
     updated_at: "Updated at %{time}"
+  user_analytics:
+    hypertension: 'Hypertension'
+    diabetes: 'Diabetes'
+    genders: ['all', male', 'female', 'transgender']
```

Comment:
Will we be able to exclude `transgender` from, say, Ethiopia's progress tab?

Reply:
I don't exactly know how that will play out. But I believe the valid gender list will be a model level concern and thereby the presence of the gender translation should not be relevant to what data needs to be displayed.

Reply:
Great point.